components/frontend/src/semantic_ui_react_wrappers/Form.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { act, render } from '@testing-library/react';
+import { DarkMode } from '../context/DarkMode';
+import { Form } from '../semantic_ui_react_wrappers';
+
+it('shows the form dropdown in darkmode', async () => {
+    await act(async () => {
+        const { container } = render(
+            <DarkMode.Provider value={true}>
+                <Form>
+                    <Form.Dropdown options={[{ key: "Hi", value: "Hi", text: "Hi" }]} value={"Hi"} />
+                </Form>
+            </DarkMode.Provider>
+        )
+        expect(container.querySelector(".inverted")).not.toBe(null)
```

Comment:
Why not just check true? Or is the logic on any value

Reply:
If I recall correctly it's not a boolean flag and setting it to false gives errors.